packages/server/src/types.ts

```diff
@@ -61,6 +61,11 @@ export type ServerOptions = {
     | Promise<Locker>
     | ((req: http.IncomingMessage) => Locker | Promise<Locker>)
 
+  /**
+   * Disallow termination for finished uploads.
+   */
+  disableTerminationForFinishedUploads?: boolean
```

Comment:
Please add this to the public documentation.

Reply:
Done